Code-the-Dream-School/dd-prac-team1-front
# Devops/add chakra (completes PRA-15)

## One Line Description
Add Chakra (documentation [here](https://chakra-ui.com/getting-started))

## Requirements
Convert all html to Chakra components

## Notes
* This is slightly different than what's in the video I recorded because I had to modify to make sure it worked with multiple routes. [Here](https://chakra-ui.com/docs/components/link#usage-with-routing-library) is the documentation I referenced in the process for how to add route links using Chakra.

## Test Steps
* Pull down this branch
* On your computer, delete your `.node_modules` and your `package-lock.json`
* Run `npm install`
* Run both the back end and the front end
* Ensure that app loads as expected

## UI/UX Outcome
localhost:3000/
<img width="747" alt="Screen Shot 2023-07-22 at 1 05 53 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/69873477/25826a6a-b0d2-4302-a36e-b38f64c65dc3">

localhost:3000/home
<img width="747" alt="Screen Shot 2023-07-22 at 1 06 06 PM" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/69873477/0af96f67-b53c-4a5e-8bbe-c96f48423bdd">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
